app/components/Edit/EditServices.js

```diff
@@ -5,26 +5,47 @@ class EditServices extends Component {
 	constructor(props) {
 		super(props);
 
-		this.state = {};
+		this.state = {
+			services: {},
+			existingServices: props.services.map((service) => {
+				let newService = service;
+				newService.key = service.id;
+				return newService;
+			}),
+			uuid: -1
```

Comment:
Keeping track of a unique key for new services. Purely for the key attribute of added components.